Sources/App/Utilities/Request+AuthorizedUser.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Request+AuthorizedUser.swift
+//  App
+//
+//  Created by Arkadiusz Żmudzin on 13/03/2020.
+//
+
+import Vapor
+import JWT
+
+extension Request {
+    var token: String {
+        if let token = self.http.headers[.authorization].first?.replacingOccurrences(of: "Bearer ", with: "") {
```

Comment:
`self.http.headers[.authorization].first?.replacingOccurrences(of: "Bearer ", with: "") ?? ""`